README.md

```diff
@@ -1,24 +1,58 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Greetings! (Back End)
 
-Things you may want to cover:
+>  API for greetings messages
 
-* Ruby version
+### Built using
+    - Ruby
+    - Ruby On Rails
+    - Rubocop
+    - PostgreSQL
+    - Github
+    - RubyMine
 
-* System dependencies
+## Getting Started
 
```

Comment:
- [ ] It would be great if you also mention where users can find the corresponding front-end repo in the backend readme and provide links to the front-end repo as well, what do you think? 🤔